playwright/pages/LoginPage.ts

```
@@ -22,14 +24,37 @@ export class LoginPage {
   }
 
-  //Requires you to be on the front page of "Info Portal"
   async LoginToAccessManagement(pid: string) {
     await this.clickLoginToAccessManagement();
     await this.authenticateUser(pid);
   }
 
+  async loginWithUserInA3(testUser: string) {
+    let lastError: unknown;
+    for (let attempt = 1; attempt <= 3; attempt++) {
+      try {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)